.gitignore

```diff
@@ -1,3 +1,6 @@
 /target/
 /tmp/
 /www/
+.lnd
```

Comment:
Where does the `.lnd` file come from?

Reply:
I added it in case users (myself included) put a .lnd file in their agora dir. Somehow, the Docker image has to import the lnd files (macaroon & tls) on launch.